### PR TITLE
Fix #4072 - OGM type and caseS direct link

### LINKED ISSUE #4072: Optical mapping type and link
When only optical mapping exists for a case, it would be nice if the caseS list had a direct link to a corresponding view (FSHD) and an appropriate sequencing modality tag (`BNG`?).
Screenshot 2023-09-06 at 10 40 12

### PR DESCRIPTION
**Pull request by dnil**
This PR adds a functionality or fixes a bug.

<details>
<summary>Testing on cg-vm1 server (Clinical Genomics Stockholm)</summary>

**Prepare for testing**
1. Make sure the PR is pushed and available on [Docker Hub](https://hub.docker.com/repository/docker/clinicalgenomics/scout-server-stage)
1. Fist book your testing time using the Pax software available at [https://pax.scilifelab.se/](https://pax.scilifelab.se). The resource you are going to call dibs on is `scout-stage` and the server is `cg-vm1`.
1. `ssh <USER.NAME>@cg-vm1.scilifelab.se`
1. `sudo -iu hiseq.clinical`
1. `ssh localhost`
1. (optional) Find out which scout branch is currently deployed on cg-vm1: `podman ps`
1. Stop the service with current deployed branch: `systemctl --user stop scout.target`
1. Start the scout service with the branch to test: `systemctl --user start scout@<this_branch>`
1. Make sure the branch is deployed: `systemctl --user status scout.target`
1. After testing is done, repeat procedure at [https://pax.scilifelab.se/](https://pax.scilifelab.se), which will release the allocated resource (`scout-stage`) to be used for testing by other users.
</details>

<details>
<summary>Testing on hasta server (Clinical Genomics Stockholm)</summary>

**Prepare for testing**
1. `ssh <USER.NAME>@hasta.scilifelab.se`
1. Book your testing time using the Pax software. `us; paxa -u <user> -s hasta -r scout-stage`. You can also use the WSGI Pax app available at [https://pax.scilifelab.se/](https://pax.scilifelab.se).
1. (optional) Find out which scout branch is currently deployed on cg-vm1: `conda activate S_scout; pip freeze | grep scout-browser`
1. Deploy the branch to test: `bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-tool-stage.sh -e S_scout -t scout -b <this_branch>`
1. Make sure the branch is deployed: `us; scout --version`
1. After testing is done, repeat the `paxa` procedure, which will release the allocated resource (`scout-stage`) to be used for testing by other users.
</details>

**How to test**:
1. how to test it, possibly with real cases/data

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
- [x] code approved by CR
- [x] tests executed by DN

### CODE REVIEW

File: scout/constants/case_tags.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
-ANALYSIS_TYPES = ("wgs", "wes", "mixed", "unknown", "panel", "panel-umi", "external", "wts")
+ANALYSIS_TYPES = ("external", "mixed", "ogm", "panel", "panel-umi", "unknown", "wes", "wgs", "wts")
```
Comment (northwestwitch) [inline scout/constants/case_tags.py:1]: 👍🏻

File: scout/server/blueprints/institutes/templates/overview/cases.html [hunk 2]
```diff
@@ -164,6 +164,8 @@
           <a href="{{ url_for('variants.cancer_sv_variants', institute_id=case.owner, case_name=case.display_name, variant_type='clinical') }}">Clinical Somatic Structural Variants</a>
         {% elif case.vcf_files.vcf_fusion %}
           <a href="{{ url_for('variants.fusion_variants', institute_id=case.owner, case_name=case.display_name, variant_type='clinical') }}">Clinical Fusion Variants</a>
+        {% elif case.individuals|rejectattr('bionano_access', 'undefined')|rejectattr('bionano_access', 'none')|list|length > 0 %}
+          <a href="{{ url_for('cases.bionano', institute_id=institute._id, case_name=case.display_name) }}">Optical Genome Mapping BioNano FSHD</a>
```
Comment (northwestwitch) [inline scout/server/blueprints/institutes/templates/overview/cases.html:168]: 💯

**Review by northwestwitch — APPROVED** [review]
Looks good! 👍🏻

**Comment by dnil** [comment]
Screenshot 2024-01-16 at 09 25 29

**Pull request merged**